k8s/nginx-3scale/nginx-3scale-svc.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ngx-instance-0
+  namespace: default
+  labels:
+    name: ngx-instance-0
+  annotations:
+    # NOTE: the following annotation is a beta feature and
+    # only available in GCE/GKE and Azure as of now
```

Comment:
Maybe include a link to an explanation in the comments as well, e.g.

https://kubernetes.io/docs/tutorials/services/source-ip/